subprojects/docs/src/docs/userguide/dependencyManagementTerminology.adoc

```diff
@@ -0,0 +1,44 @@
+[[dependency_management_terminology]]
+== Dependency Management Terminology
+
+Dependency management comes with a wealth of terminology. Here you can find the most commonly-used terms including references to the user guide to learn about their practical application.
+
+[[sub:terminology_module]]
```

Comment:
Let us make all these H3 headers so users can navigate directly to them in the secondary navigation. 

I also think alphabetical order is appropriate so terms can even more easily be found.

Reply:
done